packages/expo/src/Expo.fx.tsx

```diff
@@ -2,17 +2,9 @@
 import './winter';
 import 'expo-asset';
 
-import * as Font from 'expo-font';
-import { StyleSheet } from 'react-native';
-
 import { isRunningInExpoGo } from './environment/ExpoGo';
 import { createErrorHandler } from './errors/ExpoErrorManager';
 
-// If expo-font is installed and the style preprocessor is available, use it to parse fonts.
-if (StyleSheet.setStyleAttributePreprocessor) {
-  StyleSheet.setStyleAttributePreprocessor('fontFamily', Font.processFontFamily);
```

Comment:
This function remaps fontFamily names to sandboxed versions, which is needed in Expo Go and development clients. https://github.com/facebook/react-native/blob/9cd83fa9ab118943c4bd63e8b284c72a33ddac9b/packages/react-native/Libraries/StyleSheet/StyleSheet.js#L306-L331

Reply:
That's no longer true, we removed scoping from `expo-font` since it's been adding a huge maintenance cost and was a reason of many bugs. Related PRs: #28797 (Android), #28344 (iOS)

`processFontFamily` is no-op now https://github.com/expo/expo/blob/03f8dd512dadb9cd723ff46e58a5224a73bfc8ea/packages/expo-font/src/Font.ts#L24-L28

We left this function in `expo-font` for backwards compatibility, but it can be removed in SDK 53. It seems fine to remove this code from the `expo` package though.